crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -799,11 +882,19 @@ static void TestOperation(const EVP_CIPHER *cipher, bool encrypt,
   // |EVP_EncryptFinal_ex| and |EVP_DecryptFinal_ex| for approval at the end.
```

Comment:
Sorry, this may have come up previously. The comment says we test 
> |EVP_EncryptFinal_ex| and |EVP_DecryptFinal_ex| for approval at the end.
But I don't see we're calling Init and Update to test that they won't set Approved, then call Final to test that it does set it. Is that done elsewhere?
Also, see my comment above in the case of xts particularly where update should return approved, unless we force the FIPS user to call final which would be a NOP but just to return Approved (to be tested that this is what will happen).

Reply:
Yes this test was when of the first versions of the service indicator tests, so we hadn't checked for non-approval for `Init` and `Update`. "Taffer" has added a new non-approval check for `Update` on Line 116, but we should add one for `init` as well.